.github/ci/environment.yaml

```diff
@@ -0,0 +1,34 @@
+name: base
+
+channels:
+  - conda-forge
+
+dependencies:
+  - anaconda-client
+  - cmake
+  - colcon-common-extensions
+  - compilers
+  - cppzmq
+  - curl
+  - eigen
+  - ffmpeg
+  - freeimage
+  - gdal
+  - git
+  - glib
+  - gts
+  - jsoncpp
+  - libzip
+  - make
+  - ninja
+  - ogre
+  - pkg-config
+  - protobuf
+  - pybind11
+  - qt
```

Comment:
If you do not depend on qt-webengine anywhere, we can switch to the qt-main package.